src/codeu/chat/server/Server.java

```diff
@@ -128,6 +140,10 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
 
         Serializers.INTEGER.write(out, NetworkCode.GET_USERS_RESPONSE);
         Serializers.collection(User.SERIALIZER).write(out, users);
+      if (type == NetworkCode.SERVER_INFO_REQUEST) {
```

Comment:
See my above comment - you only need to make that change once, so you can probably delete this section.